.github/workflows/matrix-all.yml

```diff
@@ -0,0 +1,30 @@
+name: Build Matrix (All Versions)
+
+on:
+  workflow_call:
+    outputs:
+      matrix:
+        description: "Generates the all versions build matrix"
+        value: ${{ jobs.all_versions.outputs.matrix }}
+
+jobs:
+  all_versions:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: SCM Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Python & Poetry Environment
+        uses: exasol/python-toolbox/.github/actions/python-environment@0.20.0
+
+      - name: Generate matrix
+        run: poetry run nox -s matrix:all
+
+      - id: set-matrix
+        run: |
+          echo "matrix=$(poetry run nox -s matrix:all)" >> $GITHUB_OUTPUT
```

Comment:
If I am correct, then nox task matrix:all is defined in PTB.
But still uses database version 7.1.9, which is quite outdated. 
As 7.1.30 and 8.32.0 are already available. 
Do you plan to update this? 
How can a single project override the default settings? 
I propose to support using an environment variable here and to simply set it in the workflow file.

Reply:
https://exasol.github.io/python-toolbox/main/changes/changes_0.20.0.html#added